jobs/build/appregistry/Jenkinsfile

```diff
@@ -113,37 +119,125 @@ node {
     def workDir = "${env.WORKSPACE}/workDir"
     buildlib.cleanWorkdir(workDir)
 
-    currentBuild.description = "Collecting appregistry images for ${params.BUILD_VERSION}"
-    currentBuild.displayName += " - ${params.BUILD_VERSION}"
+    def buildVersion = params.BUILD_VERSION
+
+    currentBuild.description = "Collecting appregistry images for ${buildVersion}"
+    currentBuild.displayName += " - ${buildVersion}"
 
     def skipPush = params.SKIP_PUSH
 
+    def validate = { params ->
```

Comment:
I used lambdas because most functions needed to access `buildlib` or `commonlib`, and those are not available outside `node { }`. Also because with lambdas I no longer needed to whitelist stuff on the groovy sandbox, nor use the `@NonCPS` decorator.

but I agree that 3 of them don't need anything I mentioned above, were just kept as lambdas for consistency with the rest, and could just be converted to regular functions:
* `validate`
* `replaceNVRsByFoundInAdvisories`
* `pushToOMPS`

Reply:
Makes sense.